foladipo/docs-cabinet-cp2
# Feature/148218577/make documents editable

#### What does this PR do?

This PR enables a user to update his/her documents.

#### Description of task to be completed

Enable a user to update documents.

#### How should this be manually tested?

- Follow the instructions stated in this project's README to set up your local instance of this app.

- Consequently, you should have sign up for a new account.

- Open the side menu and click the `"My documents"` option. You will then be taken to a page with a list of documents.

- Click the "update" button (colored green) to update a document.

#### Any background context you want to provide?

N/A

#### What are the relevant Pivotal Tracker stories?

148218577

#### Screenshots

<img width="1279" alt="screen shot 2017-07-09 at 9 28 28 pm" src="https://user-images.githubusercontent.com/26963575/27997307-a1522268-64ed-11e7-9c71-01d23dd43c47.png">
